spec/controllers/sessions_controller_spec.rb

```diff
@@ -140,4 +140,28 @@
       end
     end
   end
+
+  describe 'DELETE#destroy' do
+    it 'returns a redirect response' do
```

Comment:
How about the non-login user access to this action?

Reply:
Everything will work as `destroy` method describe. But yes, unauthenticated user should not be able to access to this action. I will make the change.

Reply:
Fixed in c7b4c98946e036157413fa55101b29c93f4c04c5